### PR TITLE
dotnet 3.1 Quick start SQS

### PR DESCRIPTION
**Pull request by ulili5**
*Issue #112  , if available:*

*Description of changes:*
This contribution adds QuickStart SQS project for .NET Core 3.1. Since currently there are few QuickStart projects for .NET, I was inspired on the NodeJS QuickStart SQS project and I built this one to provide the same boilerplate project for .NET Developers.

Follow these steps to test this PR :

1.	Make sure you’ve .NET Core 3.1.* installed see [Download .NET Core 3.1]( https://dotnet.microsoft.com/download/dotnet/3.1)
2.	Run the following commands

```bash
python3 -m venv venv
pip install --upgrade pip aws-sam-cli
pip install -r requirements.txt
dotnet --info
sam --version
pytest -vvv tests/integration/build_invoke/test_build_invoke_dotnetcore3_1.py -n 4
pytest -vvv tests/integration/unit_test/test_unit_test_dotnetcore3_1.py -n 4
``` 

And check if the project cookiecutter-aws-sam-quick-start-s3-dotnet was able to run.

**Note:** The unit test of this project seams to works only on Linux or Mac. please do not try to test the commands showed on Windows machine;

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: dotnetcore3.1/cookiecutter-aws-sam-quick-start-sqs-dotnet/{{cookiecutter.project_name}}/events/event-sqs.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
```
Comment (mndeveci) [inline dotnetcore3.1/cookiecutter-aws-sam-quick-start-sqs-dotnet/{{cookiecutter.project_name}}/events/event-sqs.json:1]: Event file seems different then the one in Nodejs init template. https://github.com/aws/aws-sam-cli-app-templates/blob/master/nodejs14.x/cookiecutter-quick-start-sqs/%7B%7Bcookiecutter.project_name%7D%7D/events/event-sqs.json

File: dotnetcore3.1/cookiecutter-aws-sam-quick-start-sqs-dotnet/{{cookiecutter.project_name}}/src/{{cookiecutter.project_name}}/aws-lambda-tools-defaults.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
+  "Information": [
+    "This file provides default values for the deployment wizard inside Visual Studio and the AWS Lambda commands added to the .NET Core CLI.",
+    "To learn more about the Lambda commands with the .NET Core CLI execute the following command at the command line in the project root directory.",
+    "dotnet lambda help",
+    "All the command line options for the Lambda command can be specified in this file."
+  ],
+  "profile": "",
+  "region": "",
```
Comment (mndeveci) [inline dotnetcore3.1/cookiecutter-aws-sam-quick-start-sqs-dotnet/{{cookiecutter.project_name}}/src/{{cookiecutter.project_name}}/aws-lambda-tools-defaults.json:9]: Ditto

**Review by mndeveci — COMMENTED** [review]
Thanks a lot for your contribution, left 2 minor comments.

File: dotnetcore3.1/cookiecutter-aws-sam-quick-start-sqs-dotnet/{{cookiecutter.project_name}}/events/event-sqs.json (hunk 1 above)
Comment (ulili5) [reply dotnetcore3.1/cookiecutter-aws-sam-quick-start-sqs-dotnet/{{cookiecutter.project_name}}/events/event-sqs.json:1]: @mndeveci this is the correct one that SQS send, the node one content only one message, but SQS should send record:[], my .net code is parsing the message using a class provided by AWSSDK .NET which is the same message if you generate sam local 
```bash
sam local generate-event sqs receive-message
```

File: dotnetcore3.1/cookiecutter-aws-sam-quick-start-sqs-dotnet/{{cookiecutter.project_name}}/src/{{cookiecutter.project_name}}/aws-lambda-tools-defaults.json (hunk 2 above)
Comment (ulili5) [reply dotnetcore3.1/cookiecutter-aws-sam-quick-start-sqs-dotnet/{{cookiecutter.project_name}}/src/{{cookiecutter.project_name}}/aws-lambda-tools-defaults.json]: @mndeveci Done!

**Review by mndeveci — APPROVED** [review]
Thanks for handling all the feedback! 🎉

**Pull request merged**